### ISSUE TITLE
Not clear how to run code

### ISSUE THREAD

**Issue by wmatern** [mengqvist/DNApy#45]
I was interested in trying your plasmid editor as I recently got a new computer (and I do a fair bit of molecular biology for work). Unfortunately I had trouble installing/getting it running. Could you update your instructions to make sure we are able to run your code?

$ git clone https://github.com/mengqvist/DNApy.git

$ cd DNApy

$ python2 main.py
python2: can't open file 'main.py': [Errno 2] No such file or directory

$ python2 setup.py build
Traceback (most recent call last):
  File "setup.py", line 26, in <module>
    with io.open('HISTORY.md', 'r', encoding='utf-8') as f:
IOError: [Errno 2] No such file or directory: 'HISTORY.md'

I'm looking forward to this project if it is still going. I think a lot of people would benefit from a FOSS tool for molecular biology. Thank you for starting this!

**Comment by openpaul** [comment]
At them moment I do not think it is actually run able but maybe I am wrong @mengqvist?

**Comment by mengqvist** [comment]
Yes, that's right. The code is currently broken. I'm working on fixing it and making an installer etc.


On Aug 17, 2017 10:04, "Paul" <notifications@github.com<mailto:notifications@github.com>> wrote:

At them moment I do not think it is actually run able but maybe I am wrong @mengqvist<https://github.com/mengqvist>?
